fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -56,6 +81,7 @@ def run
 
       global_option('--verbose') { $verbose = true }
       global_option('--capture_output', 'Captures the output of the current run, and generates a markdown issue template') { $capture_output = true }
+      global_option('--troubleshoot', 'Enables extended verbose mode, beware this eventually includes passwords, should not be run on CI')
```

Comment:
For the description: `Enables extended verbose mode. Use with caution, as this even includes ALL sensitive data. Cannot be used on CI.`